base/applications/mscutils/eventvwr/lang/de-DE.rc

```diff
@@ -3,7 +3,7 @@
  * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:     German resource file
  * TRANSLATORS: Copyright 2007-2008 Daniel Reimer <reimer.daniel@freenet.de>
- *              Copyright 2021 Joachim Henze <joachim.henze@reactos.org>
+ *              Copyright 2021-2024 Joachim Henze <joachim.henze@reactos.org>
```

Comment:
You did a lot of translation fixes etc. in this file, didn't you? :joy:

Reply:
I don't think so, and it wasn't me who added me to that line most likely. It was very likely Stasm. So What else shall I do than playing that game and update those comments now whenever I touch the file?